Task_TheFlexGame/index.css

```diff
@@ -1 +1,61 @@
 /* Start coding below for FLEX GAME TASK*/
+
+.main {
+  display: flex;
+  flex-direction: column;
+  height: 100%;
+}
+
+.header {
+  background-color: red;
+  padding-bottom: 20px;
+  padding-left: 10px;
+  padding-top: 10px;
+  height: 100%;
+  flex: 1 1 0;
+  width: 460px;
+}
+
+.content {
+  display: flex;
+  height: 100%;
+  width: 470px;
+}
+.menu {
+flex-basis: 20%;
+display: flex;
+flex-direction: column;
+gap: 10px; 
+background-color: purple;
+height: 400px;
+padding: 30px;
+}
+
+.menuButton {
```

Comment:
Add a margin property to .menuButton to create some spacing between this element and the elements around it.